autonat.go

```diff
@@ -230,11 +240,17 @@ func (as *AmbientAutoNAT) scheduleProbe() time.Duration {
 			untilNext = as.config.retryInterval
 		} else if currentStatus.Reachability == network.ReachabilityPublic && as.lastInbound.After(as.lastProbe) {
 			untilNext *= 2
+		} else if currentStatus.Reachability != network.ReachabilityPublic && as.lastInbound.After(as.lastProbe) {
+			untilNext /= 5
+		}
+		if as.confidence < 3 && peer != "" {
+			untilNext /= 5
 		}
```

Comment:
So we try the peer if we haven't recently probed? Can we add some documentation to this function explaining why we're doing what we're doing?

Reply:
yep. added documentation to the function.